services/worker/src/worker/dtos.py

```diff
@@ -221,6 +221,7 @@ class IsValidResponse(TypedDict):
 
 DatasetTag = Literal["croissant"]
 DatasetLibrary = Literal["mlcroissant", "webdataset", "datasets", "pandas", "dask"]
+DatasetFormat = Literal["json", "csv", "parquet", "imagefolder", "audiofolder", "webdataset", "text"]
```

Comment:
we cannot discriminate between json and jsonl, right?

Reply:
We could differentiate but it requires to read some data. For now I'd keep it under "json"